gobblin-data-management/src/main/java/org/apache/gobblin/data/management/conversion/hive/avro/AvroSchemaManager.java

```diff
@@ -161,6 +161,12 @@ else if (serdeParameters != null && serdeParameters.containsKey(HiveAvroSerDeMan
         List<TypeInfo> typeInfos = fields.stream().map(fs -> TypeInfoUtils.getTypeInfoFromTypeString(fs.getType()))
             .collect(Collectors.toList());
         List<String> comments = fields.stream().map(fs -> fs.getComment()).collect(Collectors.toList());
+        // TODO: Fix this code. Error - the below method is deprecated in Schema.Field in avro 1.9
+        // java.lang.NoSuchMethodError: org.apache.avro.Schema$Field.<init>(Ljava/lang/String;Lorg/apache/avro/Schema;Ljava/lang/String;Lorg/codehaus/jackson/JsonNode;)V
+        // at org.apache.hadoop.hive.serde2.avro.TypeInfoToSchema.createAvroField(TypeInfoToSchema.java:76)
+        // at org.apache.hadoop.hive.serde2.avro.TypeInfoToSchema.convert(TypeInfoToSchema.java:61)
+        // TypeInfoToSchema is from hive-serde-1.0.1 and uses avro 1.7. The latest version 3.1.2
+        // still uses avro 1.7
```

Comment:
This is a problem, but maybe we can copy the TypeInfoToSchema class and customize it so that it uses the compat helper. Not great, but I think the blast radius will be small and manageable.